_posts/2018-04-03-bug-bbq-how-to-be-involved.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+permalink: 2018-04-03-bug-bbq-involved
+subheadline: "What can I do during the Bug BBQ?"
+title: "How to be involved in the Bug BBQ?"
+teaser: "A short guide on how to contribute and be involved during The Carpentries Bug BBQ."
+header:
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["François Michonneau"]
+---
+
+
+## I want to contribute to the lessons during the Bug BBQ!
+
+We created a guide that outlines [how to contribute](https://hackmd.io/s/r1EDctqcG#) to the Bug BBQ. There are many ways to get involved, the issues to address are as varied as the skills they require, and the time it takes to fix them.
+
+As indicated in the guide, we strongly recommend that you leave a comment on the issue you are going to address before starting to work on it. This ensures that no work gets duplicated.
+
+We will use Slack to communicate and coordinate. Feel free to ask questions in the `#general` channel or in the channel associated with the lesson you are working on. 
+
+
+## I'm a maintainer, what do I need to know before the Bug BBQ?
+
+We suggest you coordinate with your co-maintainers to decide who will be online when. If at least one member of your lesson can check-in every few hours to review pull requests, provide feedback on the issues that are being opened, and monitor the Slack channels that would be great. If you can spread the load of this work among the maintainers that makes it easier. We [prepared a spreadsheet](https://docs.google.com/spreadsheets/d/1FRsvOara8gTHqHEHoVNi9nBhsk2UXSb2tBmxHCIO1D8/edit?usp=sharing) for you to complete so that you can indicate your availability. This spreadsheet has tabs for Geospatial, Social Sciences, and all the other lessons that are part of the event.
+
+We also suggest that you prepare your repository for the Bug BBQ. Close stale issues and pull requests, assign at least one of each "type" and "status" GitHub label to the relevant issues, open issues for things you know need to be addressed. Don't forget to add the "good first issue" and "help wanted" labels to the relevant issues to make it easier for new contributors to identify things they can work on relatively easily.
+
+Don't hesitate to contact [François](mailto:francois@carpentries.org) or [Erin](mailto:ebecker@carpentries), or leave a message on Slack if you have any questions. If there is anything we can do to make it easier for you to participate in the Bug BBQ, let us know.
+
+
+## I'm a maintainer, what do I need to do during the Bug BBQ?
+
+During the Bug BBQ, as a Maintainer, you will provide feedback on the issues and pull requests that will come to your repository. You will assign GitHub labels to the issues that are being opened and update them as needed. You will answer questions contributors may have about the lesson.
```

Comment:
Add: "This is a big job! Please coordinate with your co-Maintainers to decide who will do what, and don't hesitate to contact us if we can help provide support for your team."